package.json

```diff
@@ -30,5 +30,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-router-dom": "^5.1.2"
```

Comment:
I don't think it's a dev dependency 🧐

Reply:
Thx, I'll reinstall it as normal dependency :)